tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Tests/RegexRecognizerTests.cs

```diff
@@ -92,40 +98,57 @@ public async Task RegexRecognizerTests_Intents()
             ValidateColorIntent(result);
         }
 
-        private static void ValidateColorIntent(RecognizerResult result)
+        [Fact]
+        public async Task RegexRecognizerTests_Intents_LogsTelemetry_WithLogPiiTrue()
```

Comment:
Consider making it a theory test and consolidate

Reply:
Done. Consolidated as theory